org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/cancellation/RestRequestCancellationRegistry.java

```diff
@@ -46,27 +45,22 @@ protected ConcurrentMap<String, RequestCancellationInfo> getRequestCancellationI
     return m_requestCancellationInfos;
   }
 
-  public IRegistrationHandle register(String requestId, Object userId, RunContext runContext) {
+  /**
+   * Rregister request with id {@code requestId} in registry using given {@code runContext} and {@code userId}.
```

Comment:
```suggestion
   * Register request with id {@code requestId} in registry using given {@code runContext} and {@code userId}.
```

Maybe we should also add a description that manual unregistration is possible/necessary?

Reply:
Added into javadoc